crates/bevy_ecs/src/archetype.rs

```diff
@@ -25,6 +25,11 @@ impl ArchetypeId {
         ArchetypeId(0)
     }
 
+    #[inline]
+    pub const fn invalid() -> ArchetypeId {
+        ArchetypeId(usize::MAX)
+    }
+
```

Comment:
I don't think it would actually change anything perf wise (maybe reduce a little compile time?), but I think it would be more readable if we have
```rust
impl ArchetypeId {
    pub const EMPTY: ArchetypeId = ArchetypeId(0);
    pub const RESOURCE: ArchetypeId = ArchetypeId(1);
    pub const INVALID: ArchetypeId = ArchetypeId(usize::MAX);
}
```
instead of the now three methods

Reply:
Good call!